op-node/rollup/driver/step.go

```diff
@@ -37,9 +37,19 @@ func (d *outputImpl) createNewBlock(ctx context.Context, l2Head eth.L2BlockRef,
 	// case we need to fetch all transaction receipts from the L1 origin block so we can scan for
 	// user deposits.
 	if l2Head.L1Origin.Number != l1Origin.Number {
+		if l1Origin.ParentHash != l2Head.L1Origin.Hash {
```

Comment:
comparison is opposite above comparison

Reply:
I'm not sure if I understand your comment. If the l1 origin changes by number, and the parent of the new origin does not match the previous origin, then it is some inconsistent L1 chain reference and should thus error. That should be correct right?

Reply:
Oh I see, you mean the order of the operands, not the equality check itself.

Reply:
first comparison is `l2Head != l1Origin`, second is `l1Origin != l2Head`. The order of the arguments is different in each.

Reply:
Fixed (and fixed the fix, now all three comparisons match)